python/spacewalk/common/rhnLog.py

```diff
@@ -95,8 +95,8 @@ def initLOG(log_file="stderr", level=0):
     log_path = os.path.dirname(log_file)
     if log_file not in ('stderr', 'stdout') \
             and log_path and not os.path.exists(os.path.dirname(log_file)):
-        log_stderr("WARNING: log path not found; attempting to create %s" %
-                   log_path, sys.exc_info()[:2])
+        log_stderr("WARNING: log path not found; attempting to create %s for %s" %
```

Comment:
May be you could try to set the component as a prefix even in those errors: this would make it easier to aggregate in log analyzer. So something like `"{} WARNING: log path not found; attempting to create {}".format(component, log_path)` could be better

Reply:
and yeah, it is considered better in python to use "{} foo".format(param) than the "%" notation. See https://realpython.com/python-string-formatting/ for example